Pod/Classes/LSTracer.h

```diff
@@ -59,6 +59,9 @@ extern NSInteger const LSBackgroundTaskError;
 /// `LSTracer` instance's globally unique id ("guid"), and assigned automatically by LightStep.
 @property(nonatomic, readonly) UInt64 runtimeGuid;
 
+/// HTTP session to be used for performing requests. This enable sharing a connnection pool with your own app.
```

Comment:
Thanks for catching, updating. One side thought - it might be worth to introduce a value object like `LSConfiguration` and have only one initializer instead of all the permutations.
This way you could make the properties immutable and safer